src/o365/spo/SpoCommand.ts

```diff
@@ -124,4 +127,110 @@ export default abstract class SpoCommand extends Command {
         });
     });
   }
+
+  public ensureFormDigest(cmd: CommandInstance, context: FormDigestInfo, debug: boolean): Promise<FormDigestInfo> {
+    return new Promise<FormDigestInfo>((reject: (error: any) => void): void => {
+      if (this.isUnexpiredFormDigest(context)) {
+
+        if (debug) {
+          cmd.log('Existing form digest still valid');
+        }
+
+        Promise.resolve(context);
```

Comment:
Calling `Promise.resolve` doesn't stop the code flow, so this method will continue to execute event though it shouldn't. To truly stop it, the method should call `return`. Compare to the original implementation to see the difference.